extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -108,3 +79,39 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	return true;
 }
+
+function fetchReviewHistory( $dbr, $cityId, $pageId ) {
+	// TODO: fetch history from image
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)